src/modules/login/Login.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+import { TEMPLATE_ERROR_STRING } from "@modules/registration/Registration";
+import "../registration/index.scss";
```

Comment:
For future, please don't use styles from outer modules. You can do it by using specific components, but using file with styles may cause issue in the future when you need to change something only in that specific place.